### PR TITLE
fix(aiplatform): Fix doc formatting

### LINKED ISSUE #358: Docstring for a few of the methods are not parsable
Docstring for a few of the methods under aiplatform are not parsable, which breaks API doc generation and parsing tools. For example in training_jobs.py the docstring 

```python
    def _raise_failure(self):
        """Helper method to raise failure if TrainingPipeline fails.

        Raises:
            RuntimeError If training failed.
        """
```
should be changed to 
```python 
    def _raise_failure(self):
        """Helper method to raise failure if TrainingPipeline fails.

        Raises:
            RuntimeError: If training failed.
        """
```

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Fixing docstring formatting for aiplatform.(Fixes #358)

### CODE REVIEW

File: google/cloud/aiplatform/helpers/value_converter.py [hunk 1]
```diff
@@ -21,7 +21,8 @@
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/helpers/value_converter.py:21]: Please revert this file.

File: google/cloud/aiplatform/utils.py [hunk 2]
```diff
@@ -318,14 +320,14 @@ def __init__(
...
 
-            client_class (AiPlatformServiceClient):
-                Required. Class of the client to use.
-            client_options (client_options.ClientOptions):
-                Required. Client options to pass to client.
-            client_info (gapic_v1.client_info.ClientInfo):
-                Required. Client info to pass to client.
-            credentials (auth_credentials.credentials):
-                Optional. Client credentials to pass to client.
+            client_class (AiPlatformServiceClient):     Required. Class
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/utils.py:323]: It looks like this one needs to be indented with an "Args:" section as it was missing it initially.

File: google/cloud/aiplatform/utils.py [hunk 3]
```diff
@@ -367,12 +369,11 @@ def __init__(
     ):
         """Stores parameters needed to instantiate client.
 
-        client_options (client_options.ClientOptions):
-            Required. Client options to pass to client.
-        client_info (gapic_v1.client_info.ClientInfo):
-            Required. Client info to pass to client.
-        credentials (auth_credentials.credentials):
-            Optional. Client credentials to pass to client.
+        client_options (client_options.ClientOptions):     Required.
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/utils.py:372]: Same comment as above.

**Review by sasha-gitg — CHANGES_REQUESTED** [review]

File: google/cloud/aiplatform/helpers/value_converter.py (hunk 1 above)
Comment (SinaChavoshi) [reply google/cloud/aiplatform/helpers/value_converter.py]: done.

**Review by sasha-gitg — APPROVED** [review]
Thanks Sina!

**Pull request merged**